src/fasta-recs.py

```diff
@@ -11,7 +12,50 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
+    # Read the content
+    content = args.fasta.read()
+    
+    index = 0
+
+    while(index<len(content)):
+        # Look for '> *'
+        match = re.search(">(\s)*", content[index:])
+        if match is None:
+            break
+        index += match.end()
+
+        beginName = index
+
+        # Look for the end of the name
+        match = re.search("\n", content[index:])
+        if match is None:
+            chain = content[beginName:len(content)]
+            chain = re.sub('(\s)+',' ',chain).strip()
+            print(chain, end="\t")
+            break
+        index += match.start()
+
+        chain = content[beginName:index]
+        chain = re.sub('(\s)+',' ',chain).strip()
+        print(chain, end="\t")
+
+        beginChain = index        
+        # Look for the genome chain
```

Comment:
The genome is actually the whole file. What you are extracting here is a chromosome. (Not important for the code, of course, but could be confusing for the reader of the comment).

Reply:
Okay, I take note of your comments and I'll try to fix them tomorrow. Thanks :)